cl/corpus_importer/management/commands/harvard_opinions.py

```diff
@@ -159,17 +159,14 @@ def parse_harvard_opinions(reporter, volume):
         # same page.  So we use page count as a proxy for it.
         pg_count = 1 + int(data["last_page"]) - int(data["first_page"])
         if cite_search.count() > 0:
-            cluster = cite_search[0].cluster
-            if cluster.date_filed is not None:
-                date_filed, is_approximate = validate_dt(
-                    data['decision_date'])
-                if cluster.date_filed != date_filed:
-                    logger.info("Duplicate cite string different date filed")
-                elif cluster.page_count != pg_count:
-                    logger.info("Duplicate cite string but diff page count")
-                else:
-                    logger.info("%s Already in CL." % cite)
-                    continue
+            cluster_ids = cite_search.values_list("cluster_id")
+            objects = OpinionCluster.objects.filter(
+                id__in=cluster_ids).values_list('case_name')
```

Comment:
Calling a variable `objects` is a no-go. But beyond that, there's hardly ever a reason to look up items by ID like this. I rewrote these.